### PR TITLE
Bugfix: Migrator shunt does not work with '/*' mapping

### PR DESCRIPTION
**Pull request by mdegerne**
[#156242036]

### CODE REVIEW

File: test/integration/test_migrator.py [hunk 1]
```diff
@@ -290,6 +298,57 @@ def _check_container_created():
...
+                        return False
+                    else:
+                        raise
+                swift_names = set([obj['name'] for obj in listing])
+                done = done and (test_names == swift_names)
+                if not done:
+                    return False
+            return True
+
+        # self.swift_nuser2.put_container('thisisstupid')
```
Comment (dbishop) [inline test/integration/test_migrator.py]: dead code?

**Comment by mdegerne** [comment]
Removed the comment.

File: test/container/swift-s3-sync.conf [hunk 2]
```diff
@@ -204,6 +204,14 @@
             "aws_secret": "\u062apass2",
             "container": "no-auto-history",
             "protocol": "swift"
+        },
+        {
+            "account": "AUTH_nacct2",
```
[the remark this answers is not in the document]
Comment (mdegerne) [reply test/container/swift-s3-sync.conf]: n is for new. I was having difficulty parsing out for my own sanity what bits referred to accounts and what bits to users so this helped me. I'm sorry for using icky names.

File: test/container/launch.sh [hunk 3]
```diff
@@ -52,6 +54,12 @@ if ! grep -q "$tempauth_line1" /etc/swift/proxy-server.conf; then
     sed -i "s/egg:swift#tempauth/&\n$tempauth_line1\n$tempauth_line2/" /etc/swift/proxy-server.conf
 fi
 
+tempauth_nuser=$'user_nacct_nuser = npass .admin'
+tempauth_nuser2=$'user_nacct2_nuser2 = npass2 .admin'
+if ! grep -q "$tempauth_nuser" /etc/swift/proxy-server.conf; then
+    sed -i "s/user_test_tester3 = testing3/&\n$tempauth_nuser\n$tempauth_nuser2/" /etc/swift/proxy-server.conf
```
[the remark this answers is not in the document]
Comment (dbishop) [reply test/container/launch.sh]: Yeah, all this mutation of the proxy-server.conf should be replaced with a full copy in `test/container/` that then gets copied into the container in `launch.sh`.

File: test/container/swift-s3-sync.conf (hunk 2 above)
[the remark this answers is not in the document]
Comment (dbishop) [reply test/container/swift-s3-sync.conf]: ahahahahahhaa, if you think nacct is bad, you should see _my_ account names, @clayg 😝

**Pull request merged**